setup.py

```diff
@@ -33,14 +37,25 @@
     license="MIT",
     classifiers=[
         "License :: OSI Approved :: MIT License",
+        "Intended Audience :: Developers",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
     ],
-    packages=find_packages(),
+    python_requires=">=3.7",
+    packages=find_packages(
+        include=["stytch*"],
+        exclude=["*.test", "*.tests", "*.test.*", "*.tests.*"],
+    ),
     include_package_data=True,
     install_requires=[
+        "aiohttp>=3.8.3",
```

Comment:
[pebble] Is there a way to make this dependency optional? It might be annoying for an app to have this extra dependency if they're not using the async API.

Reply:
Oooooh, that could be fun. Let me see if I can make an extra_features like [async] so if you just want a standard client you do "stytch~=6.0" in your requirements.txt, but for async, you do "stytch[async]~=6.0"

I'll play around with that.